lib/common/logging.c

```diff
@@ -838,7 +838,10 @@ crm_log_preinit(const char *entity, int argc, char *const *argv)
 
     have_logging = true;
 
-    pcmk__xml_init();
+    /* @TODO Try to create a more obvious "global Pacemaker initializer"
+     * function than crm_log_preinit(), and call pcmk__schema_init() there
+     */
```

Comment:
https://projects.clusterlabs.org/T840 is relevant here.

Reply:
Added a note to the comment